docs/api.md

```diff
@@ -16,20 +16,21 @@ contributions, no restrictions and the most valuable ones will be integrated int
 Here is the most basic skeleton for creating a new `GEF` command named `newcmd`:
 
 ```python
+@register
 class NewCommand(GenericCommand):
     """Dummy new command."""
     _cmdline_ = "newcmd"
     _syntax_  = f"{_cmdline_}"
+    # optionally
+    # _examples_ = [f"{_cmdline_}", ]
+    # _aliases_ = [f"alias1", ]
```

Comment:
```suggestion
    _examples_ = [f"{_cmdline_} ...", ]
    _aliases_ = ["alias1", ]
```

Reply:
`_examples_` and `_aliases_` are optional, but I re-arranged it a bit